src/ctap/data_formats.rs

```diff
@@ -432,6 +440,27 @@ impl TryFrom<&cbor::Value> for SignatureAlgorithm {
     }
 }
 
+#[derive(Clone, PartialEq, PartialOrd)]
```

Comment:
You can make this enum `Copy` as well. This doesn't take a lot of bytes in memory, and allows simpler code elsewhere (see my other comment).

Reply:
Done.